server/metrics.go

```diff
@@ -0,0 +1,35 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/openfresh/plasma/config"
+	"github.com/openfresh/plasma/metrics"
+)
+
+func NewMetricsServer(config config.Config, metrics []metrics.Metrics) *http.Server {
```

Comment:
__[golint]__
* [golint] exported function NewMetricsServer should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->